torch/lib/THD/base/data_channels/DataChannelGloo.cpp

```diff
@@ -20,9 +15,12 @@
 
 
 #define RETURN_IF_NOT_IN_GROUP(name)                                          \
-  rank_type ##name; bool _e;                                                  \
-  std::tie(##name, _e) = _groups.at(group_id).getGroupRank(_rank);            \
-  if (!_e) return;
+  rank_type name;                                                             \
+  {                                                                           \
+    bool exists;                                                              \
+    std::tie(name, exists) = _groups.at(group_id).getGroupRank(_rank);        \
+    if (!exists) return;                                                      \
+  }
```

Comment:
You are right, it is not used anymore. Thanks!

It is not necessary in our case, I guess. It is not used in weird places so this should be problem. I used parenthesis to create block in which `bool exists` variable will be local.